pkg/cmd/migrate.go

```diff
@@ -45,6 +46,41 @@ func runMigration(_ *cobra.Command, _ []string) error {
 	goose.SetLogger(goose.NopLogger())
 
 	switch engine {
+	case "mysql":
+		db, err := sql.Open("mysql", uri)
+		if err != nil {
+			log.Fatal("failed to parse the config from the connection uri", err)
+		}
+
```

Comment:
Sorry! Something I just realized we forgot for Postgres too is the following. Could you please add this here too? Thank you! 

```
defer func() {
	if err := db.Close(); err != nil {
		log.Fatal("failed to close the db", err)
	}
}()
```

Reply:
Done!